app/components/floating_text_input_label/index.tsx

```diff
@@ -142,25 +74,15 @@ const FloatingTextInput = forwardRef<FloatingTextInputRef, FloatingTextInputProp
     const debouncedOnFocusTextInput = debounce(setIsFocusLabel, 500, {leading: true, trailing: false});
     const styles = getStyleSheet(theme);
 
-    const positions = useMemo(() => getLabelPositions(styles.textInput, styles.label, styles.smallLabel), [styles]);
+    const hasStartAdornment = Boolean(startAdornment);
+    const hasEndAdornment = Boolean(endAdornment);
+    const positions = useMemo(() => getLabelPositions(styles.textInputContainer, styles.label, styles.smallLabel), [styles]);
     const size = useMemo(() => [styles.textInput.fontSize, styles.smallLabel.fontSize], [styles]);
-
-    useImperativeHandle(ref, () => ({
-        blur: () => inputRef.current?.blur(),
-        focus: () => inputRef.current?.focus(),
-        isFocused: () => inputRef.current?.isFocused() || false,
-    }), [inputRef]);
-
-    useEffect(
-        () => {
-            if (!focusedLabel && value) {
-                debouncedOnFocusTextInput(true);
-            }
-        },
-        [value],
-    );
+    const textInputContainerHeight = useMemo(() => getInputContainerHeight(multiline, textInputContainerStyle), [multiline, textInputContainerStyle]);
+    const textInputHeight = useMemo(() => textInputContainerHeight - (INPUT_CONTAINER_VERTICAL_SPACING * 2), [textInputContainerHeight]);
 
     const onTextInputBlur = useCallback((e: NativeSyntheticEvent<TextInputFocusEventData>) => onExecution(e,
+        // eslint-disable-next-line max-nested-callbacks
```

Comment:
Are these happening with the current config, or because of some local config you have? If it has to do with some local config, it would be great if you could remove these disable.

Reply:
![Screenshot 2022-11-16 at 15 50 32](https://user-images.githubusercontent.com/17684951/202228071-2d6531cb-778d-4ed0-b081-84eaebf0a3b7.png)
This is configured on eslint-mattermost.json

Reply:
you can extract the function to avoid so many nested callbacks